ttnn/cpp/ttnn/operations/pool/upsample/upsample.cpp

```diff
@@ -11,6 +11,7 @@ namespace ttnn::operations::upsample {
 
 ttnn::Tensor ExecuteUpSample::invoke(const ttnn::Tensor& input_tensor,
     std::variant<int, tt::tt_metal::Array2D, tt::tt_metal::Array3D, tt::tt_metal::Array4D> scale_factor,
+    std::string mode,
     std::optional<MemoryConfig> output_mem_config) {
         MemoryConfig mem_config = output_mem_config.value_or(ttnn::DRAM_MEMORY_CONFIG);
         int scale_h = 1;
```

Comment:
It is best not to hard code the ttnn::DRAM_MEMORY_CONFIG.   Can we pull the value from the input_tensor instead when not provided?

Reply:
Done